src/main/java/no/nav/familie/integrasjoner/client/rest/KodeverkClient.kt

```diff
@@ -38,6 +42,14 @@ class KodeverkClient(@Value("\${KODEVERK_URL}") private val kodeverkUri: URI,
         return UriUtil.uri(kodeverkUri, "api/v1/kodeverk/$kodeverksnavn/koder/betydninger", query)
     }
 
+    private fun httpHeaders(): HttpHeaders {
+        return HttpHeaders().apply {
+            contentType = MediaType.APPLICATION_JSON
+            accept = listOf(MediaType.APPLICATION_JSON)
+            add("Nav-Consumer-Token", "Bearer ${stsRestClient.systemOIDCToken}")
```

Comment:
Hvorfor?

Reply:
Denne PR var for å teste en sak, men fant ut nå att den blir sendt med fra StsBearerTokenClientInterceptor